langs/groovy/Dockerfile

```diff
@@ -42,7 +42,7 @@ COPY --from=0 /groovy-$VER.jar                              \
               /usr/lib/libskarnet.so.2.14                   \
               /usr/lib/libutmps.so.0.1                      /usr/lib/
 
-ENV JAVA_HOME=/opt/java
+ENV JAVA_HOME='/opt/java'
```

Comment:
Just curious, I've noticed you doing this a lot lately. Why quote something that doesn't need quoting?